README.md

```diff
@@ -44,6 +45,11 @@ Some links relevant to users and developers of WTMO:
   [WTMO Developer Guide](https://mana.mozilla.org/wiki/display/DOPS/WTMO+Developer+Guide)
   (behind SSO)
 
+## Contributing to this Repo
+For someone without write access to the repo, the best way to create a Pull Request is to fork this repo, make your changes in the forked repo, and submit a pull request from the forked repo into the central repository.  More info on how to do this can be found [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).  In order to merge a PR, someone with write access must approve it.
+
+PR name and description must follow [Conventional Commit style](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
```suggestion
For contributors without write access to the repo, the best way to create a Pull Request is to fork this repo, make your changes in the forked repo, and submit a pull request from the forked repo into the central repository.  More info on how to do this can be found [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).  In order to merge a PR, someone with write access must approve it.

Most contributors to this repo are likely to have write access. For these contributors, using a fork is not necessary and creating a branch in this repo directly is the preferred workflow. The normal PR process still applies.

For all PRs (forked or not), PR name and description must follow [Conventional Commit style](https://www.conventionalcommits.org/en/v1.0.0/).
```